server/models/Profile.js

```diff
@@ -28,12 +28,16 @@ const profileSchema = new mongoose.Schema({
   },
   birthday: {
     type: Date,
-    default: null
+    default: null,
   },
   photo: {
     type: String,
     default: "",
   },
+  accountType: {
```

Comment:
We can adjust the base of this PR/rebase in order to only reflect changes pertaining to this feature